### DISCUSSION TITLE
[oracle-adf-faces-miracle-rce] [http] [high]  --again stupidity

### DISCUSSION THREAD

**Discussion by ghost** [projectdiscovery/nuclei-templates#8962] [Q&A] [answered]
hi Team 
I can see some issue .Nuclei show my in some website some wrong outcome .This is second website were I can see wrong outcome.
[oracle-adf-faces-miracle-rce] [http] [high]

https://taurus-systems.de/de/kontakt/bicomposer/afr/khewz/remote/H4sIAAAAAAAACnWMsU7DMBCGr6RRU9QBGBg6IrHaLZRAYSgSUkWksIDYcRu3NjKxsa_FXXgMnoKXQDwBKzMrb4AETpWVX7rTd9L93%2Bs3xM7C9j1bMrJAqcglc%2BKKmbj1%2Bfa%2Be_cRwcYYNpVmxZhNUdsM2igsd0KrwpvROVTpPCVhb4VpBFlnLSs5ktvr_GX_OO3%2BfNkIWhkkIsgvdMEzaBptMYc2W6DQVuIKYSevilSxck5v0MpyfpZDcyYVf4RniAIL7bDmxFiNeqpVfUeWz9bozW8dhK6Q2Ds4TAe94SDtD4_6JxMy8c6RB47QoNV_jEGLaIxH2KvglNL_W9T_AbBHMtQxAQAA/

**Comment by ehsandeep** [comment]
@kolor33 there is no template with id `oracle-adf-faces-miracle-rce` in template project, what template / issue you referring here?

**Comment by ghost** [comment]
This can be this but Nuclei do not show CVE number 
nuclei-templates/http/cves/2022/CVE-2022-21445.yaml 

https://github.com/projectdiscovery/nuclei-templates/issues/8794

**Comment by ehsandeep** [comment]
@kolor33 none of them part of template project, if you are running templates from GitHub issues, make sure to know what you are running, they are not reviewed or validated by the template team.

**Comment by olearycrew** [comment, marked as answer]
@kolor33 yes that template hasn't been accepted yet - it will need more accurate matchers as currently it will match any `500` status

**Comment by ghost** [comment]
Thanks and sorry .I was in too much of a hurry
